src/org/thoughtcrime/securesms/database/RecipientDatabase.java

```diff
@@ -197,7 +201,8 @@ public BlockedReader readerForBlocked(Cursor cursor) {
                                              RegisteredState.fromId(registeredState),
                                              profileKey, systemDisplayName, systemContactPhoto,
                                              systemPhoneLabel, systemContactUri,
-                                             signalProfileName, signalProfileAvatar, profileSharing));
+                                             signalProfileName, signalProfileAvatar, profileSharing,
+                                              nickName));
```

Comment:
Is there a line limit in the coding style guide?

Reply:
no